.github/workflows/cmake.yml

```
@@ -163,7 +168,13 @@ jobs:
       - name: Set environment variables
         run: |
           echo "COLUMNS=79" >> ${GITHUB_ENV}
-          echo "LD_LIBRARY_PATH=$PWD/install/lib" >> ${GITHUB_ENV}
+          if [ "${{ matrix.ubsan }}" == "ON" ]; then
```

```suggestion
          if [ "${GRN_WITH_UBSAN}" = "ON" ]; then
```

* In general, environment variable is better than `${{ ... }}` because we can unify evaluation phase. (`${{ ... }}` is expanded BEFORE `run` is evaluated.)
* In general, `=` is better than `==` for `[` because `=` is POSIX compatible.

---

Thank you so much!